ISSUE_TEMPLATE.md

```diff
@@ -13,3 +14,10 @@
 ```
 <paste it here between the little tick marks>
 ```
+
+**Environment (select all that apply):**
```

Comment:
Perhaps just ask users to upload a copy of their core.yml via a gist or hastebin instead. This would help identify configuration problems (SQL or otherwise).

The other don't seem necessary to me. Reloads are known to be frowned upon in the community, and server owners are aware it can cause problems in plugins. Offline mode has no impact on LWC and should be left out of the issue template. The latest server version should already be included above under "Server version (/version)".